dissent-oracle/README.md

```diff
@@ -1,9 +1,47 @@
-# Dissent Oracle
+# Dissent Oracle <img align="right" src="https://github.com/1Hive/website/blob/master/website/static/img/bee.png" height="80px" />
 
-Oracle implementation installed along with the Dandelion Voting app to prevent engagement when a user has recently voted in favour of a proposal.
+The Dissent Oracle app is an [ACL Oracle](https://hack.aragon.org/docs/acl_IACLOracle). ACL Oracles are small helper functions that plug in to Aragon's access control list (ACL) to do more sophisticated permission evaluation. The Dissent Oracle is installed along with the Dandelion Voting app to prevent engagement when a user has recently voted in favour of a proposal.
 
-## Usage
+#### 🐲 Project stage: development
 
-Applied to an Aragon app's permission's params.
+The Time Lock app is still in development, a first implementation was published to `time-lock.open.aragonpm.eth`. If you are interested in contributing please see our open [issues](https://github.com/1hive/time-lock-app/issues).
```

Comment:
This is for the dissent oracle not for time-lock no?

Reply:
Yup. Lots of copypasta for the READMEs and lots of link errors. Thanks for catching that. 

Also, FTR created a README template in devops to address that in the future: https://github.com/1Hive/devops/blob/master/docs/README.md